ci/tests/fputil.bash

```diff
@@ -17,7 +17,7 @@ export FPUTIL_TARGETS=("generate" "generate --ut" "build" "build --all" "check -
 function fputil_action {
     export WORKDIR="${1}"
     export TARGET="${2}"
-    let JOBS="${JOBS:-$(( ( RANDOM % 9 )  + 1 ))}"
+    let JOBS="${JOBS:-$(( ( RANDOM % 100 )  + 1 ))}"
```

Comment:
Consistent with what was prior to https://github.com/nasa/fprime/pull/2437
Also wasn't modified here: https://github.com/nasa/fprime/pull/2466/files#diff-f2defb2d2bbc3a725bff91bb51af54316c775cb3ccbf549f21a77d12411ee08bR46